components/esp_lvgl_port/README.md

```diff
@@ -0,0 +1,145 @@
+# LVGL ESP Portation
+
+This component helps with using LVGL with Espressif's LCD and touch drivers. It can be used with any project with LCD display. 
+
+## Features
+* Initialization of the LVGL
+    * Create task and timer
+    * Handle rotating
+* Add/remove display (using [`esp_lcd`](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/lcd.html))
+* Add/remove touch input (using [`esp_lcd_touch`](https://github.com/espressif/esp-bsp/tree/master/components/lcd_touch))
+
+## Usage
+
+### Initialization
+```
+    const lvgl_port_cfg_t lvgl_cfg = ESP_LVGL_PORT_INIT_CONFIG();
+    esp_err_t err = lvgl_port_init(&lvgl_cfg);
+```
+
+### Add screen (IDF 5.1)
+
+Add an LCD screen to the LVGL. It can be called multiple times for adding multiple LCD screens. 
+```
+    /* LCD driver initialization */
+    ...
+    esp_lcd_panel_handle_t lcd_panel_handle;
+    esp_err_t err = esp_lcd_new_panel_st7789(io_handle, &panel_config, &lcd_panel_handle);
+
+    /* Add LCD screen */
+    const lvgl_port_display_cfg_t disp_cfg = {
+        .io_handle = io_handle,
+        .panel_handle = lcd_panel_handle,
+        .buffer_size = DISP_WIDTH*DISP_HEIGHT,
+        .double_buffer = true,
+        .hres = DISP_WIDTH,
+        .vres = DISP_HEIGHT,
+        .monochrome = false,
+        /* Rotation values must be same as used in esp_lcd for initial settings of the screen */
+        .rotation = {
+            .swap_xy = false,
+            .mirror_x = false,
+            .mirror_y = false,
+        },
+        .flags = {
+            .buff_dma = true,
+        }
+    };
+    lv_disp_t * disp = lvgl_port_add_disp(&disp_cfg);
+```
+
+**Note:** The screens added in this function are not removed in `lvgl_port_deinit`. They must be removed by `lvgl_port_remove_disp` before deinitialization. Otherwise, there can be memory leaks!
+
+### Add screen (IDF 5.0 and IDF 4.4)
```

Comment:
It took me a while to spot the differences between IDF v5.1 and this chapter. 

Would it be better if we highlighted only the code that is different? That is, only the lcd_io configuration part?

Reply:
Agree, but I don't know, how to highlight for good visibility changes?

Reply:
Sorry, poor wording on my side :D

I was thinking about leaving the lcd_io configuration part only in this chapter. And have a comment like: 'The rest of the configuration is the same as in IDF v5.1'